ext/nnstreamer/tensor_filter/tensor_filter_llama2.cc

```diff
@@ -165,17 +182,24 @@ TensorFilterLlama2c::invoke (const GstTensorMemory *input, GstTensorMemory *outp
 
   result = get_tokens (transformer, tokenizer, sampler, prompt, steps);
 
-  /** @todo Make out_info flexible */
-  output[0].size = tokenizer->max_token_length * steps;
+  if (!result) {
```

Comment:
Q: is it possible if returned token is empty ?
```
if (!result || (len = strlen (result)) == 0)
```

Reply:
`result`'s memory is allocated in llama2c, so it must be freed even if length is 0.
I added a logic to check if string is empty!
Also, I found `prompt`'s memory is not relased by your review.
Thank you :)